src/components/InputWithLabel.module.css

```diff
@@ -5,3 +5,10 @@ label {
 input {
   margin: 1em;
 }
+@media only screen and (max-width: 600px) {
+  /*label {
```

Comment:
Is this intentionally commented out? If so, I would recommend removing it (and the CSS rule as a whole) since it doesn't style anything in your app.

Reply:
Merged with PR https://github.com/umavenki/react-todo/pull/13